frontend/src/components/Wizard/WizardMetadata.vue

```diff
@@ -3,102 +3,108 @@
     <v-select
       v-model="metadataLanguage"
       variant="outlined"
-      :label="$t('metadataLanguage')"
-      required
+      :label="t('metadataLanguage')"
+      :rules="SomeItemSelectedRule"
       item-title="DisplayName"
       item-value="TwoLetterISOLanguageName"
-      :items="cultureOptions" />
+      :items="cultureOptions"
+      :disabled="loading" />
     <v-select
       v-model="metadataCountry"
       variant="outlined"
-      :label="$t('metadataCountry')"
-      required
+      :label="t('metadataCountry')"
+      :rules="SomeItemSelectedRule"
       item-title="DisplayName"
       item-value="TwoLetterISORegionName"
-      :items="countryOptions" />
+      :items="countryOptions"
+      :disabled="loading" />
     <v-btn
       color="secondary"
       variant="elevated"
-      @click="$emit('previous-step', { step: 2 })">
-      {{ $t('previous') }}
+      :disabled="loading"
+      @click="emit('previous-step')">
+      {{ t('previous') }}
     </v-btn>
     <v-btn
       :loading="loading"
       color="primary"
       variant="elevated"
       @click="setMetadata">
-      {{ $t('next') }}
+      {{ t('next') }}
     </v-btn>
   </div>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
 import {
   CountryInfo,
   CultureDto,
   StartupConfigurationDto
 } from '@jellyfin/sdk/lib/generated-client';
 import { getLocalizationApi } from '@jellyfin/sdk/lib/utils/api/localization-api';
 import { getStartupApi } from '@jellyfin/sdk/lib/utils/api/startup-api';
-import { useSnackbar } from '@/composables';
+import { useRemote, useSnackbar } from '@/composables';
 
-export default defineComponent({
-  setup() {
-    return {
-      useSnackbar
-    };
-  },
-  data() {
-    return {
-      metadataLanguage: '',
-      metadataCountry: '',
-      initialConfig: {} as StartupConfigurationDto,
-      cultureOptions: [] as CultureDto[],
-      countryOptions: [] as CountryInfo[],
-      loading: false
-    };
-  },
-  async created() {
-    const api = this.$remote.sdk.oneTimeSetup(
-      this.$remote.auth.currentServer?.PublicAddress || ''
-    );
+const emit = defineEmits<{
+  (e: 'step-complete'): void;
+  (e: 'previous-step'): void;
+}>();
 
-    this.initialConfig = (
-      await getStartupApi(api).getStartupConfiguration()
-    ).data;
+const SomeItemSelectedRule = [(v: unknown[]): boolean | string => v.length > 0];
```

Comment:
At this point I think we can extract these rules to something like ``utils/validation-rules.ts`` instead of duplicating them all over the place?

For now I think extracting this one is good, but we can look into doing the same thing for the rules used at the login pages.